transformations/font_change/transformation.py

```diff
@@ -0,0 +1,79 @@
+import itertools
+import json
+import os
+import random
+import re
+
+from nltk import download as nltkdl
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+nltkdl("stopwords")
+nltkdl("punkt")
+
+
+def font_change(sentence, fonts, seed=666, max_outputs=1):
+    random.seed(seed)
+    perturbed_texts = []
+
+    for _ in itertools.repeat(None, max_outputs):
+        # Generate the sentence with random fonts
+        tokens_escaped_list = []
+        for token in word_tokenize(sentence):
+            if token not in stopwords.words("english"):
+                if token != ".":
+                    tokens_escaped_list.extend(
+                        list(re.finditer(re.escape(token), sentence))
+                    )
+
+        transformed_sentence = list(sentence)
+
+        tokens_to_change = random.sample(
+            tokens_escaped_list,
+            random.randint(1, min(3, len(tokens_escaped_list) - 1)),
+        )
+        for ttc in tokens_to_change:
+            while True:
+                font = random.sample(list(fonts.keys()), 1)[0]
+                if font != "normal":
```

Comment:
Why not just removing "normal" from the dictionary in the JSON file?

Reply:
It was by choice. I did not want any discrepancy between my dictionary and the original resource. Of course, it can be removed.